DEBIAN/postrm

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+case "$1" in (remove)
+	rm /usr/bin/openrefine /usr/share/applications/openrefine.desktop
+	rm -rf /opt/openrefine	
+	rm -rf $HOME/.openrefine
```

Comment:
Should we really be deleting user data here? I think this should only happen with the `--purge` option.